README.md

```diff
@@ -1,44 +1,50 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# :construction: IAPPS Web App Front-End
 
-## Available Scripts
+This repo contains the setup for IAPPS Frontend.
 
-In the project directory, you can run:
+## DO NOT ADD ANY EXTERNAL LIBRARIES
 
-### `npm start`
+Everything needed to develop the project has been setup properly including routing, css libraries, liniting files, build process. DO NOT add any external library without discussion. Please simply take a pull from develop and start adding HTML, CSS and TypeScript Code.
 
-Runs the app in the development mode.<br>
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+## Collaboration Guidelines :checkered_flag:
 
-The page will reload if you make edits.<br>
-You will also see any lint errors in the console.
+### Code Quality :art:
 
-### `npm test`
+* No class based Components. Use [hooks](https://reactjs.org/docs/hooks-overview.html#state-hook) only.
+* No implicit types. To ensure the code is of the highest quality, use explicit types when declaring interfaces.
+* Always develop the individual component in isolation using [React Storybook](https://storybook.js.org/). Do not add components directly to app. Check for all possible inputs and scenarios in storybook first.
+* Always use [code splitting](https://facebook.github.io/create-react-app/docs/code-splitting) using ```React.lazy``` and ```React.suspense```
```

Comment:
There should be a guide on how to split code using these. A blog post would do nicely.

Reply:
Yep. We could write on your blog. Can you guide me on how would I go about doing it? for example should I add code snapshots with explanations?

Reply:
Write something first. We'll review it later.